src/styles/lines/lines.js

```diff
@@ -9,13 +9,17 @@ import {buildPolylines} from '../../builders/polylines';
 import Geo from '../../geo';
 import Utils from '../../utils/utils';
 
+let fs = require('fs');
+let shaderSrc_polygonsVertex = fs.readFileSync(__dirname + '/../polygons/polygons_vertex.glsl', 'utf8');
```

Comment:
Yeah. It's more of a bikeshedding philosophical choice, as `lines` shouldn't depend on `polygons` so it shouldn't import it.